src/kernel/arch/aarch64/tty.zig

```diff
@@ -111,11 +147,29 @@ pub fn writeString(str: []const u8) !void {
     }
 }
 
+///
+/// Set the text cursor
+///
+/// Arguments:
+///     IN x: usize - The x coordinate (the text column, starting at 0 on the left)
+///     IN y: usize - The y coordinate (the text row, starting at 0 at the top)
+///
 pub fn setCursor(x: u8, y: u8) void {
     framebuffer.text_cursor_x = x;
     framebuffer.text_cursor_y = y;
 }
 
+///
+/// Create and initialize a TTY object for the frame buffer
+///
+/// Arguments:
+///     IN allocator2: *std.mem.Allocator - not yet used
+///     IN board: arch.BootPayload - determines what 
```

Comment:
determines what?

Reply:
Corrected comment and now using the allocator that is passed.